src/Term/TermFallback.php

```diff
@@ -46,6 +46,18 @@ public function __construct( $requestedLanguageCode, $text, $actualLanguageCode,
 			throw new InvalidArgumentException( '$sourceLanguageCode must be a non-empty string or null' );
 		}
 
+		if ( $actualLanguageCode === $sourceLanguageCode ) {
```

Comment:
But the other way round, why should anybody use this? What does `new TermFallback( 'en', 'text', 'en' )` express?